packages/splitter/splitter.stories.tsx

```diff
@@ -193,30 +200,139 @@ Hook.parameters = {
   }
 };
 
-export const Controlled = ({
+export const ManyHooks = ({
   type,
   min,
   max,
   orientation,
   ariaLabel,
-  primaryPaneId,
+  defaultValueNow,
   keyboardStep,
   rtl
 }) => {
   const Splitter = () => {
-    const [valueNow, onChange] = useState(300);
     const mode = rtl ? 'rtl' : 'ltr';
-    const { getSeparatorProps } = useSplitter({
+    const firstSplitterBag = useSplitter({
```

Comment:
`firstSplitterBag`? What do you mean by bag?

Reply:
It's a convention borrowed from Formik for object containing library methods and properties  https://formik.org/docs/api/withFormik#the-formikbag